src/Stays/StaysTypes.ts

```diff
@@ -123,9 +123,10 @@ export interface StaysRate {
   id: string
 
   /**
-   * The accepted payment method for this rate. Prepaid rates require payment at time of reservation. Accepted types: pay_now, guarantee
-   */
-  payment_type: 'pay_now' | 'guarantee'
+   * The accepted payment method for this rate. Prepaid rates require payment at time of reservation.
+   * Deposit rates require a deposit payment, then the remainder at the accommodation.
+   * */
+  payment_type: 'pay_now' | 'guarantee' | 'deposit'
```

Comment:
Why are these duplicated here? Shouldn't they come from @duffel/api?

Reply:
...isn't this @duffel/api? if not then i might be misinformed as to how this works 😅

Reply:
this seems to stem from a very old misnomer, and this may be the chance to fix it.

in the beginning of this file, there is an export of StaysPaymentTypes, which ends up in @duffel/api. it mistakenly holds payment methods.

if we rename it to be StaysPaymentMethods, make these the actual StaysPaymentType, and line up everything else that uses those types, we may be able to namespace these enum types properly